ansible/vagrant/vagrant-ansible.yml

```diff
@@ -1,19 +1,20 @@
 - hosts: all
   sudo: yes
+  vars:
+    - public_iface: eth0
```

Comment:
That fact won't be defined here though, so it'll be a little more of a pain to get it right...